scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -459,4 +458,15 @@ protected void selectionChangedHandler(SelectionEvent event) {
         }
     }
 
+    @Override
+    protected void selectionUpdatedHandler(SelectionUpdatedEvent event) {
+        ListGridRecord[] selectedRecords = this.getSelectedRecords();
+        ArrayList<String> selectedTasksIds = new ArrayList<>();
```

Comment:
Same here you should specify the length of the array list when you can. It will avoid to reallocate memory.